src/components/vm/nics/nicBody.jsx

```diff
@@ -75,16 +79,35 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
     let networkSourceEnabled = true;
 
     // { name: 'ethernet', desc: 'Generic ethernet connection' }, Add back to the list when implemented
+    const virtualNetwork = [{
+        name: 'network',
+        desc: 'Virtual network',
+        detailHeadline: _("This is the recommended config for general guest connectivity on hosts with dynamic / wireless networking configs."),
+        detailParagraph: _("Provides a connection whose details are described by the named network definition.")
+    }];
     if (connectionName !== 'session') {
         availableNetworkTypes = [
-            { name: 'network', desc: 'Virtual network' },
-            { name: 'bridge', desc: 'Bridge to LAN' },
-            { name: 'direct', desc: 'Direct attachment' },
+            ...virtualNetwork,
+            {
+                name: 'bridge',
+                desc: 'Bridge to LAN',
+                detailHeadline: _("This is the recommended config for general guest connectivity on hosts with static wired networking configs."),
+                detailParagraph: _("Provides a bridge from the guest virtual machine directly onto the LAN. This assumes there is a bridge device on the host physical machine which has one or more of the host physical machines physical NICs enslaved.")
```

Comment:
This is very long and hard to read due to the repetitions. How about

```suggestion
                detailParagraph: _("Provides a bridge from the guest virtual machine directly onto the LAN. This needs a bridge device on the host with one or more physical NICs.")
```